src/forms/reduxForm/reduxTextInput.test.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
I'm trying to test that our wrapping ReduxTextInput component renders out a TextInput with a snapshot containing the expected props.

After working through different errors, this test needed a Form hooked up with reduxForm, and Provider. I followed this example https://github.com/erikras/redux-form/issues/849.